scripts/rollup/config.js

```diff
@@ -4,11 +4,18 @@ import babel from 'rollup-plugin-babel';
 import replace from 'rollup-plugin-replace';
 import { uglify } from 'rollup-plugin-uglify';
 import filesize from 'rollup-plugin-filesize';
+import semver from 'semver';
 
-if (process.env.NODE_ENV === 'production' && !process.env.VERSION) {
-  throw new Error(
-    'You need to specify an environment variable `VERSION` to run the build process.'
-  );
+if (process.env.NODE_ENV === 'production') {
+  if (!process.env.VERSION) {
+    throw new Error(
+      'You need to specify an environment variable `VERSION` to run the build process.'
```

Comment:
We can merge the conditions to a single with `process.env.NODE_ENV === 'production'` (like previously):

```suggestion
      'You need to specify a valid semver environment variable `VERSION` to run the build process (received ${process.env.VERSION}).'
```

Reply:
It's a good idea. I've updated it.

---

BTW, TIL `1.2.3-beta` is a valid semver.

```
> semver.valid('1.2.3-beta.')
null
> semver.valid('1.2.3-beta')
'1.2.3-beta'
> semver.valid('1.2.3-beta.1')
'1.2.3-beta.1'
```